CosmosContracts/juno-tools
# feat(deps): upgrade all dependencies

## Description

This PR upgrade all dependencies and bump to React 18.

## Changes

List any technical changes.

- [x] [upgrade all dependencies](https://github.com/CosmosContracts/juno-tools/commit/9ef5abd04442826a4f1a66000b4af192805872fe)
- [x] [resolve anchor wrap typing](https://github.com/CosmosContracts/juno-tools/commit/8e314749638a0ccfb554c9b11b82859d3cc160e6)

## Screenshots

\-

## Testing Steps

As a reviewer, what steps should I take to verify this is working correctly?

- Make sure running dev server and build script works

## Links

Add links to Figma files, documentation, etc.

\-
